website/static/js/contribAdder.js

```diff
@@ -60,11 +86,18 @@ var AddContributorViewModel = oop.extend(Paginator, {
         self.results = ko.observableArray([]);
         self.contributors = ko.observableArray([]);
         self.selection = ko.observableArray();
+
+        self.contributorIDsToAdd = ko.pureComputed(function() {
+            var selected_ids = self.selection().map(function(user){
```

Comment:
As per COSDev, use `lowerCamelCase` for variable names. That said, this variable probably isn't necessary; just return the return value of `.map`.